docs/content/bundle/create.md

```diff
@@ -0,0 +1,200 @@
+
+To use a mixin in your bundle:
+
+1. Install the mixin on your computer.
+   Follow the mixin's instructions to install it using the [porter mixin install] command.
+   
+   Mixins published by the Porter project can be installed with `porter mixin install NAME`.
+   If the mixin is published by a third party, you will need to specify the \--url or \--feed-url flags so Porter knows where to find the mixin.
+3. Add the name of the mixin to the **mixins** section in porter.yaml.
+   For example, to install the [terraform mixin], you would add an array entry with the value "terraform":
+   ```yaml
+   mixins:
+   - exec
+   - terraform
+   ```
+
+### Use a Custom Dockerfile
+
+When you are just getting started, it may be easier to start small and use a [custom Dockerfile] instead of creating your own mixins.
+By default, Porter generates a Dockerfile for your bundle image automatically.
+To use a custom Dockerfile, uncomment the **dockerfile** field in your porter.yaml file to use the template.Dockerfile in your bundle directory instead.
+
+```yaml
+dockerfile: template.Dockerfile
+```
+
+From there, anything you can do with Docker and [Buildkit], you can do in your custom Dockerfile.
+You can install tools, certificates, define environment variables, mount secrets, define build arguments, clone repositories, copy files from your local filesystem, and more.
+
+## Customize the Install Action
+
+The default install action uses the exec mixin to call the helpers.sh script in the bundle directory.
+The script prints out "Hello world".
+
+Add an array entry, called a **step**, to the **install** section in porter.yaml and use a mixin to run a command when the bundle is installed.
+The syntax for each mixin is different so reference the documentation for your particular mixin to know what to specify.
+The general form for any mixin is as follows:
+```yaml
+install:
+- exec:
+    description: Optional description of the step
+    # ... mixin specific values
+- terraform:
+    description: Optional description of the step
+    # ... mixin specific values
+```
+
+If you are not using custom mixins, put your bundle's logic in an executable, such as bash script or compiled binary, and place it in the bundle directory (next to your porter.yaml file).
+During porter build, these files are copied into the bundle's image and can be used when the bundle is run.
+You can always use the exec mixin when there isn't an existing mixin, or the mixin doesn't support a particular command that you require.
+
+🙏🏼 Please, [do not embed bash commands] directly in the porter.yaml file because it is much more difficult to get the escaping and quotes correct than putting the bash in a separate file.
+
+Bundles actions should be [idempotent], meaning an action can be repeated without causing errors, and it should have the same effect each time.
```

Comment:
Is this `[idempotent]` part supposed to reference to another link on the site? Since it isn't yet doing so.
![image](https://user-images.githubusercontent.com/7043511/163081068-9ca42a20-6cb6-4f6e-b4b0-27a2f8a74267.png)

Reply:
Thanks, that shouldn't be a link. I'll remove it.